### PR TITLE
Fix url endpoint

### PR DESCRIPTION
**Pull request by neilime**
Generated state url should be "http://bucket-.name.s3-website.region.amazonaws.com" instead of "http://bucket-.name.s3-website-region.amazonaws.com"

### CODE REVIEW

**Comment by eahefnawy** [comment]
@neilime Thanks for submitted 😊 

 I think both are correct depending on the region. Which region are you using?

Furthermore, that is likely a breaking change since it modifies the outputs. I think the best way to handle this is to output the bucket url based on the selected region. There might be more work required for this PR.

Ref: https://docs.aws.amazon.com/AmazonS3/latest/dev/WebsiteEndpoints.html

**Comment by neilime** [comment]
I'm using 'eu-west-3'.
I've proposed something to handle endpoints diff base on region

File: serverless.js [hunk 1]
```diff
@@ -101,7 +127,14 @@ class Website extends Component {
 
     this.state.bucketName = inputs.bucketName
     this.state.region = inputs.region
-    this.state.url = `http://${bucketOutputs.name}.s3-website-${inputs.region}.amazonaws.com`
+    
+    const endpointPattern = (
+      endpointsPatterns[inputs.region] 
+      || endpointsPatterns[null]
+    ).replace('%region%', inputs.region);
```
Comment (eahefnawy) [inline serverless.js]: can't we do this with a `getUrl` function and pass it a region? We then use string literals. It's more readable this way imo.

File: serverless.js [hunk 2]
```diff
@@ -10,6 +10,32 @@ const {
...
+  'ca-central-1': 's3-website.%region%.amazonaws.com',
+  'cn-northwest-1': 's3-website.%region%.amazonaws.com.cn',
+  'eu-central-1': 's3-website.%region%.amazonaws.com',
+  'eu-west-1': 's3-website-%region%.amazonaws.com',
+  'eu-west-2': 's3-website.%region%.amazonaws.com',
+  'eu-west-3': 's3-website.%region%.amazonaws.com',
+  'eu-north-1': 's3-website.%region%.amazonaws.com',
+  'sa-east-1': 's3-website-%region%.amazonaws.com',
+  'me-south-1': 's3-website.%region%.amazonaws.com',
+}
```
Comment (eahefnawy) [inline serverless.js]: awesome compilation! Thanks for that 👌

File: serverless.js (hunk 1 above)
Comment (neilime) [reply serverless.js]: More readable, you're right

**Pull request closed without merging**